### ISSUE TITLE
Training code for results on README.md?

### ISSUE THREAD

**Issue by cassianocasagrande** [keras-team/keras-applications#14]
Is it possible to share the code used to generate the data on the README? Was each model trained using the suggested hyperparameters in each paper or all training used the same set of hyperparameters (e.g. batch size, optimizer, lr)?

Thanks in advance

**Comment by taehoonlee** [comment]
@cassianocasagrande:

- Training codes are not available in `keras-applications`. Please see the original repos exhibited in README. There are the suggested hyperparameters for each paper.
- Test codes with 2012 ILSVRC ImageNet validation set (50000 images) are in my PC. I will share them as soon as possible.
